File: drs/src/test/com/db4o/drs/test/Person.java
/* Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com */

package com.db4o.drs.test;

public class Person {

	private String _name;
	private int _age;
	
	public void setName(String name) {
		this._name = name;
	}
	
	public String getName() {
		return _name;
	}

	public void setAge(int age) {
		this._age = age;
	}

	public int getAge() {
		return _age;
	}
	
	public Person(String name, int age) {
		this._name = name;
		this._age = age;
	}
}

File: drs/drs.net/drs.tests/src/com/db4o/drs/test/Person.cs
namespace com.db4o.drs.test
{
	public class Person
	{
		private string _name;

		private int _age;

		public virtual void SetName(string name)
		{
			this._name = name;
		}

		public virtual string GetName()
		{
			return _name;
		}

		public virtual void SetAge(int age)
		{
			this._age = age;
		}

		public virtual int GetAge()
		{
			return _age;
		}

		public Person(string name, int age)
		{
			this._name = name;
			this._age = age;
		}
	}
}
